lib/rubocop/cop/rspec/around_block.rb

```diff
@@ -29,19 +29,26 @@ class AroundBlock < Base
         MSG_NO_ARG     = 'Test object should be passed to around block.'
         MSG_UNUSED_ARG = 'You should call `%<arg>s.call` ' \
                          'or `%<arg>s.run`.'
+        MSG_NUMBLOCK = 'You should call `_1.call` ' \
```

Comment:
`MSG_UNUSED_ARG` covers this, if you pass `_1` as an `arg`

Reply:
Indeed it is. change to `MSG_UNUSED_ARG `